README.md

```diff
@@ -52,15 +80,15 @@ You can build the project with all supported [Mbed OS build tools](https://os.mb
     
     The above command will read your credential files and place them into a C header file for you: `aws_credentials.h`
 
-1. Once you have created the "thing", you will need to obtain the custom endpoint name from the console. At the time of writing this document, you can find it under "Settings" in the IoT console.
+3. Once you have created the "thing", you will need to obtain the custom endpoint name from the console. At the time of writing this document, you can find it under "Settings" in the IoT console.
```

Comment:
Don't change the line number. Using `1.` for every item is a special syntax (not a mistake) - Markdown automatically renumbers them, so we don't need to maintain those numbers. Same comment for number below.